opentelemetry-exporter-geneva/opentelemetry-exporter-geneva/src/logs/exporter.rs

```diff
@@ -41,13 +31,11 @@ impl fmt::Debug for GenevaExporter {
 }
 
 impl opentelemetry_sdk::logs::LogExporter for GenevaExporter {
-    /// Export logs to stdout
-    async fn export(&self, _batch: LogBatch<'_>) -> OTelSdkResult {
-        //serialize to otlp format
-        let otlp = group_logs_by_resource_and_scope(_batch, &self.resource);
-        //TODO send to Geneva using geneva-uploader
-        let _ = self.uploader.upload_logs(otlp).await;
-
+    async fn export(&self, batch: LogBatch<'_>) -> OTelSdkResult {
+        let otlp = group_logs_by_resource_and_scope(batch, &self.resource);
```

Comment:
We are forcing everyone to transform the data into the OTLP structs. This is inefficient if our user does not deal with OTLP data in memory. We could update the `GenevaClient::upload_logs` method to operate on a trait instead. We could then have different implementations for this trait: for OTLP structs, for SDK's LogBatch etc.